openverse-api/test/media_integration.py

```diff
@@ -0,0 +1,74 @@
+"""
+End-to-end API tests for audio. Can be used to verify a live deployment is
```

Comment:
Little typo here: `audio` --> `all media types` ?

Reply:
@dhruvkb let's make sure we get this one fixed separately

Reply:
Shouldn't have merged from a phone. It didn't show any comments except "approved". Fixed in #177.